README.md

```diff
@@ -40,7 +40,13 @@ with the following compilers:
 
 This library also supports C++20 coroutines. You will need to compile with
 coroutine support enabled if you want to use the coroutine integrations.
-This generally means adding `-std=c++2a` or `-fcoroutines-ts` on Clang (see "Configuring" below).
+This generally means adding `-std=c++2a`, `-fcoroutines-ts` on Clang, or
+`-fcoroutines` for GCC (see "Configuring" below). Compilers with stable
+coroutine support are required:
+
+* GCC, 10.3 and later
+* Clang, 10.x and later
```

Comment:
I'm also not sure what version of clang would be sufficient to be considered "stable" to use coroutines with libunifex, so I've left it at the minimum version of support for libunifex as stated above.